dev/integration_tests/flutter_gallery/test_driver/run_demos.dart

```diff
@@ -30,7 +30,7 @@ Future<void> runDemos(List<String> demos, WidgetController controller) async {
     final String demoName = demo.substring(0, demo.indexOf('@'));
     final String demoCategory = demo.substring(demo.indexOf('@') + 1);
     print('> $demo');
-    await controller.pump(const Duration(milliseconds: 250));
+    await controller.pump(const Duration(milliseconds: 500));
```

Comment:
I think this is the only delay that will affect "Text Fields@Cupertino". Since the previous demo is also in the "Cupertino" category, the `else if (currentDemoCategory != demoCategory)` branch won't be taken. And since "Text Fields@Cupertino" isn't in the `kUnsynchronizedDemos` list, the `if (kUnsynchronizedDemos.contains(demo))` branch won't be taken.